types/handlebars-webpack-plugin/package.json

```
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "handlebars": ">=4.7.7"
```

```suggestion
        "handlebars": ">=4.0.3",
```
Shouldn't it be supporting same minimum version?
https://github.com/sagold/handlebars-webpack-plugin/blob/master/package.json#L37

---

Oh...
Thank you so much for pointing it out, I really didn't know.
First time mistakes... 😅